### PR TITLE
Component dashboard

### PR DESCRIPTION
**Pull request by YESENIA2012**
[the modal was implemented, which contains the view profile button and sign out button.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,8 +11,10 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "git": "^0.1.5",
```
Comment (erick2014) [inline package.json]: this shouldn't be necessary, remove it and test please

File: src/Components/avatarUser/AvatarUser.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { paintAvatarAndName } from "../../utils";
+import "./avatarStyles.scss";
+
+const AvatarUser = () => {
+  const [seeModal, setSeeModal] = useState(false);
+  const [textAvatar, setTextAvatar] = useState("");
+  const [userName, setUserName] = useState("");
+
+  useEffect(() => {
+    paintAvatarAndName(setTextAvatar, setUserName);
```
Comment (erick2014) [inline src/Components/avatarUser/AvatarUser.js]: lets rename this function to `drawAvatarAndName`

File: src/utils.js [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+};
+
+/* saveUserInformation(); */
+
+const getEventsFromLocalStorage = (pageNumber) => {
+  const userInformation = getFromLocalStorage();
+  const userId = userInformation.idUser;
+  const events = JSON.parse(localStorage.getItem("Events"));
+
+  const eventsPerPage = 6;
```
Comment (erick2014) [inline src/utils.js]: let's create another function called, `filterEventsByUserId`, inside this function you can place the code from line 40 to 75 and return the events filtered. Call this new function on line 40

File: src/Components/avatarUser/AvatarUser.js (hunk 2 above)
Comment (YESENIA2012) [reply src/Components/avatarUser/AvatarUser.js]: I am going to use this function getAvatarAndName() to fetch data from local storage

File: src/utils.js (hunk 3 above)
Comment (YESENIA2012) [reply src/utils.js:40]: I used this getEventsFromLocalStorage() function to make this

File: src/Components/eventClicked/DetailEvent.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import "./detailEventStyle.scss";
+
+const DetailEvent = () => {
```
Comment (erick2014) [inline src/Components/eventClicked/DetailEvent.js]: this file should be inside events folder not here

**Pull request merged**